lib/htmlValidator.js

```diff
@@ -22,8 +24,9 @@ module.exports = function(app, callback) {
       htmlArrLength,
       htmlLine,
       formattedHTML,
+      validatorTimeout,
       errorPage = path.normalize(__dirname + '/../defaultErrorPages/views/validatorErrors'),
-      validatorDisabled;
+      validatorDisabled = true;
```

Comment:
This shouldn't need to be hardcoded. It's taken care of in the following if statement.